### ISSUE TITLE
Copyq disappears when I click on it

### ISSUE THREAD

**Issue by sohelpfuljungleboy** [hluk/CopyQ#2767]
After restart, the copyq icon appears in the lower right hand corner, along with the printer, drives, network, sound, etc, icons

if I left click on it, it just disappears. If I right click on it, I get a list of 5 clipboard entries and some program options. If I then right click on one of those options, copyq disappears.

I have to restart to get the icon back. It won't start from the menu.

When I try terminal commands, I get “Command 'copyq' not found”

I don’t think I am going to be able to recover the items.

If I uninstall via Software Manager, will the clipboard items be deleted?
If I uninstall via Software Manager, then re-install via the terminal, will the items still be available?
Is there a file somewhere that has those items?

Thanks

System info:

System:
  Kernel: 5.15.0-113-generic x86_64 bits: 64 compiler: gcc v: 11.4.0 Desktop: Cinnamon 5.6.8
    tk: GTK 3.24.33 wm: muffin dm: LightDM Distro: Linux Mint 21.1 Vera base: Ubuntu 22.04 jammy
Machine:
  Type: Desktop System: System76 product: Thelio v: thelio-r2 serial: <superuser required>
  Mobo: System76 model: Thelio v: thelio-r2 serial: <superuser required> UEFI: System76
    v: F33g Z5 date: 04/27/2021
CPU:
  Info: 8-core model: AMD Ryzen 7 5800X bits: 64 type: MT MCP arch: Zen 3 rev: 0 cache:
Drives:
  Local Storage: total: 4.09 TiB used: 339.91 GiB (8.1%)
memory: 62.7 GiB

**Comment by hluk** [comment]
> If I uninstall via Software Manager, will the clipboard items be deleted?

I don't think so, if it is installed from the official distribution repository. The data are available in `~/.config/copyq` and `~/.local/share/copyq` - feel free to back them up.

But it might not be the case if the app is installed as Flatpak or AppImage (or whatever Ubuntu can use).

> If I uninstall via Software Manager, then re-install via the terminal, will the items still be available?
Is there a file somewhere that has those items?

If the app is installed from official distro repo again, all should work as before.
